base/sparse/sparsematrix.jl

```diff
@@ -295,7 +295,7 @@ Convert a sparse matrix or vector `S` into a dense matrix or vector.
 """
 full
 
-float(S::SparseMatrixCSC) = SparseMatrixCSC(S.m, S.n, copy(S.colptr), copy(S.rowval), float(copy(S.nzval)))
+float(S::SparseMatrixCSC) = SparseMatrixCSC(S.m, S.n, copy(S.colptr), copy(S.rowval), float.(copy(S.nzval)))
```

Comment:
Maybe just `float.(S.nzval)` here?   Calling `copy` seems to result in two copies being made.